doc/Marine_wrangling.Rmd

```diff
@@ -0,0 +1,465 @@
+---
+title: "R Notebook"
+output: html_notebook
```

Comment:
Great use of comments! Makes it easy to follow along.

In general for the plot heavy coding files, consider including the .html file with a table of contents so that it is easy for your team members (and us) to quickly browse through the code and give feedback (not mandatory, just an idea).